gcloud-java-resourcemanager/src/main/java/com/google/gcloud/resourcemanager/testing/LocalResourceManagerHelper.java

```diff
@@ -61,6 +87,10 @@
   private static final Pattern LIST_FIELDS_PATTERN =
       Pattern.compile("(.*?)projects\\((.*?)\\)(.*?)");
   private static final String[] NO_FIELDS = {};
+  private static final String PERMISSION_PREFIX = "resourcemanager.projects.";
+  private static final Set<String> PERMISSION_SUFFIXES = ImmutableSet.of("create", "delete", "get",
```

Comment:
do you want to leave "create" and "list" here?

Also, you can use a static initializer to dynamically populate this list from the enum values.